state-chain/chains/src/sol/sol_tx_core.rs

```diff
@@ -1633,4 +1632,247 @@ mod tests {
 			]
 		);
 	}
+
+	// These tests can be used to manually serialize a transaction from a Solana Transaction in
+	// storage, for instance if a transaction has failed to be broadcasted. While this won't be
+	// necessary after PR#5229 it might be needed before that if we need to debug and/or manually
+	// broadcast a transaction. The serialized output (hex string) can be broadcasted via the Solana
+	// RPC call `sendTransaction` or the web3js `sendRawTransaction.
+	// The signature of the transaction is what in storage is named `transactionOutId`, since in
+	// Solana the signature is the transaction identifier.
+	#[test]
```

Comment:
We can annotate this with #[ignore] since it's not an actual unit test that we need to run in CI etc.